src/SFA.DAS.SharedOuterApi/AppStart/ConfigurationBuilder.cs

```diff
@@ -29,6 +31,12 @@ public static IConfigurationRoot BuildSharedConfiguration(this IConfiguration co
                     }
                 );
             }
+#if DEBUG
```

Comment:
This needs to be above the config.AddAzureTableStorage part, as more often than not the developer will put any of the configuration secrets for reading from azure storage in that file

Reply:
this is an interesting conundrum then, the last configuration read always wins. If our appsettings.development.json files are using mock apis or we are testing locally we need to read this file last. Otherwise we are in this awful situation of having to manually change our local azurestorage.